sdk/src/test/java/io/dapr/client/DaprHttpTest.java

```diff
@@ -215,8 +215,8 @@ public void invokePostMethodUnknownError() throws IOException {
   public void testCallbackCalledAtTheExpectedTimeTest() throws IOException {
     String deletedStateKey = "deletedKey";
     String existingState = "existingState";
-    String urlDeleteState = DaprClientHttp.STATE_PATH + "/" + deletedStateKey;
-    String urlExistingState = DaprClientHttp.STATE_PATH + "/" + existingState;
+    String urlDeleteState = STATE_PATH + "/" + deletedStateKey;
+    String urlExistingState = STATE_PATH + "/" + existingState;
```

Comment:
It seems little wierd to do the work in the tests of building a path and then splitting it again because the API takes `string[]`

Reply:
It was to minimize changes.